microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshot-schema.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Heap Snapshot Schema
+description: Use the Detached Elements tool in the Microsoft Edge DevTools to find and fix DOM memory leaks.
+author: MSEdgeTeam
+ms.author: joselea
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 9/13/2022
+---
+# Heap Snapshot Schema
+
+_In this document we will explain the schema of a `*.heapsnapshot` file which you can take and export using DevTools. This is helpful in case you want to create your own analysis tools or visualizations._
```

Comment:
Maybe try to make this a bit more compelling. Why would someone want to create their own visualizations of a heapsnapshot? Maybe give an example if possible.

Or rephrase to something like

"Investigating memory usage in web applications can be hard. The DevTools Memory tool helps with it, but you may sometimes need to focus on specific parts of the memory data that the Memory tool doesn't show. DevTools makes it possible to extract the entire memory data as a `*.heapsnapshot` file which you can then analyze and visualize. This article describes the structure and content of heapsnapshot files so you can build your own tools."